homeassistant/components/apache_kafka/__init__.py

```diff
@@ -141,7 +147,8 @@ async def shutdown(self, _: Event) -> None:
 
     async def write(self, event: Event[EventStateChangedData]) -> None:
         """Write a binary payload to Kafka."""
+        key = self._get_key(event)
```

Comment:
Is this the only place we're calling this function? If so, any reason to not inline the logic here?

Reply:
There is no reason specifically, other than this is my first time venturing into this codebase, and I wanted to be respectful. So, I was trying to match the stylistic choices of the original author. I'd be happy to inline it if that is preferred.

Reply:
Appreciate that! Go ahead and inline. 👍🏻

Reply:
Updated.